testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -5397,6 +5397,20 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkNull("array_length(null)");
   }
 
+  /** Tests {@code MAP_ENTRIES} function from Spark. */
+  @Test void testMapEntriesFunc() {
+    final SqlOperatorFixture f0 = fixture();
+    f0.setFor(SqlLibraryOperators.MAP_ENTRIES);
+    f0.checkFails("^map_entries(map['foo', 1, 'bar', 2])^",
+        "No match found for function signature MAP_ENTRIES\\(<\\(CHAR\\(3\\), INTEGER\\) "
+            + "MAP>\\)", false);
+    final SqlOperatorFixture f = f0.withLibrary(SqlLibrary.SPARK);
+    f.checkScalar("map_entries(map['foo', 1, 'bar', 2])", "[{foo, 1}, {bar, 2}]",
```

Comment:
how about `map_entries(map[])`?

Reply:
MAP_VALUE_CONSTRUCTOR: Map requires at least 2 arguments you can see code  testMapValueConstructor @JiajunBernoulli

Reply:
you can construct an empty array with the Apache Spark function `array()` I added in CALCITE-5624

Reply:
this collection is for map not array @MasseGuillaume

Reply:
haha wops 🤦. I will implement `map()` is coming soon.

Reply:
@MasseGuillaume 
and i have added the test in the pr you also refers to this problem
https://github.com/apache/calcite/pull/3212#discussion_r1209647600